apps/condo/migrations/20250124162345-0446_auto_20250124_1123.js

```diff
@@ -0,0 +1,22 @@
+// auto generated by kmigrator
+// KMIGRATOR:0446_auto_20250124_1123:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMCBvbiAyMDI1LTAxLTI0IDExOjIzCgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucwoKCmNsYXNzIE1pZ3JhdGlvbihtaWdyYXRpb25zLk1pZ3JhdGlvbik6CgogICAgZGVwZW5kZW5jaWVzID0gWwogICAgICAgICgnX2RqYW5nb19zY2hlbWEnLCAnMDQ0NV9vcmdhbml6YXRpb25lbXBsb3llZXJlcXVlc3RoaXN0b3J5cmVjb3JkX2FuZF9tb3JlJyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgIF0K
+
+exports.up = async () => {
+    const { renameRedisKeys } = require('../../../bin/rename-redis-keys')
```

Comment:
You need to ensure that this file remains functional in the future. It's unclear to future developers whether their refactoring of this binary file will disrupt the migration process.

Reply:
Yes, that's a good point